src/aliased_buffer-inl.h

```diff
@@ -126,6 +126,20 @@ void AliasedBufferBase<NativeT, V8T>::Release() {
   js_array_.Reset();
 }
 
+template <typename NativeT, typename V8T>
+inline void AliasedBufferBase<NativeT, V8T>::WeakCallback(
+    const v8::WeakCallbackInfo<AliasedBufferBase<NativeT, V8T>>& data) {
+  AliasedBufferBase<NativeT, V8T>* buffer = data.GetParameter();
+  DCHECK(buffer->is_valid());
+  buffer->cleared_ = true;
```

Comment:
The global handle should be reset by the weak callback contract.

```suggestion
  buffer->cleared_ = true;
  js_array_.Reset();
```